app/migrations/Version20250402103942.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20250402103942 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE form_fields ADD field_width VARCHAR(50) DEFAULT \'100%\' NOT NULL');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE form_fields DROP field_width');
+    }
+}
```

Comment:
I missed this migration. It's missing the preUp check and has no table prefix. Please look at how other migrations are done. I'll try to fix this.

Reply:
Fixed in https://github.com/mautic/mautic/pull/15003